multicast/send.py

```
@@ -312,6 +312,14 @@ def setupArgs(cls, parser):
 				dest="data",
 				default="PING from {name}: group: {group}, port: {port}",
 			)
+			# v2.0.9: Added the --pipe option
+			parser.add_argument(
+				"--pipe",
+				action="store_const",
+				const=["-"],
+				dest="data",
+				help="read message from stdin (equivalent to --message -)"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)